src/InstrumentationEngine/ProfilerManager.cpp

```diff
@@ -860,7 +855,7 @@ HRESULT CProfilerManager::AddInstrumentationMethod(
     CComPtr<IEnumInstrumentationMethodSettings> pSettingsEnum;
     IfFailRet(pSource->EnumSettings(&pSettingsEnum));
 
-    return this->AddInstrumentationMethod(pInstrumentationMethod.release(), pSettingsEnum, ppInstrumentationMethod);
```

Comment:
If you're removing `this->` calls can you also remove the one for `SetEventMask` on line 1111?

Reply:
Yep